config/initializers/saml_settings.rb

```diff
@@ -0,0 +1,23 @@
+module SAML
```

Comment:
Lots of changes to come here in follow-on PRs, once we figure out the devops story a bit better